src/Hydrator/EntityHydrator.php

```diff
@@ -226,6 +226,17 @@ protected function hydrateNode(Node $node, $class = null)
         return $entity;
     }
 
+    /**
+     * @param Node $node
+     * @param object $entity
+     */
+    public function refresh(Node $node, $entity)
+    {
+        $this->hydrateProperties($entity, $node);
+        $this->hydrateLabels($entity, $node);
+
```

Comment:
@kbsali I tried the cs fixer config locally. Impossible to run it with php7. I'll try to have a proper config compatible with my setup when I have some time